### ISSUE TITLE
💻 macOS support?

### ISSUE THREAD

**Issue by c-z-chen** [geekdada/flomo-cli#1]
👋 Hi there ~

I ran the following command in my command line: 

```
➜  ~ curl -sf https://gobinaries.com/geekdada/flomo-cli | sh

  ==> Downloading github.com/geekdada/flomo-cli@master
  ==> Resolved version master to v0.2.4
  ==> Downloading binary for darwin amd64
  ==> Installing flomo-cli to /usr/local/bin
  ==> Installation complete
```

however, when I run the given command (in [README](https://github.com/geekdada/flomo-cli)): 

```
flomo new --api https://flomoapp.com/foo/foo/foo/ "flomo-cli 很酷"
```

It tells me:

```
zsh: command not found: flomo
```

which means the installation was not successful, albeit it told me `==> Installation complete` formerly.

I am just wondering, after checking the releases, if there is macOS support for the fact that your script considered my client as `darwin amd64` which is kinda mistaken. 🤣

---

anyway cool project! 🕶

**Comment by geekdada** [comment]
Thank you for pointing out. It is a typo. The package name is `flomo-cli` so the binary name should be `flomo-cli`.

At first, I was thinking about providing my own pre-built binary and using `flomo` as the name of the binary, but then I found https://gobinaries.com/ . gobinaries doesn't provide a way to rename the binary, it just follow the name of the go package.

Using gobinaries to install go package is just wonderful, so I decided to change the name to `flomo-cli`, but I forgot to change README.md.
